kubetest/gke.go

```diff
@@ -171,6 +173,15 @@ func newGKE(provider, project, zone, region, network, image, imageFamily, imageP
 		return nil, fmt.Errorf("--gke-shape must include a node pool named 'default', found %q", *gkeShape)
 	}
 
+	switch subnetMode := *gkeSubnetMode; subnetMode {
+	case "auto", "custom":
+		g.subnetMode = subnetMode
+	case "legacy":
```

Comment:
NIT: Let's delete this, saying that we support either 'custom' or 'auto' is enough.

Reply:
+1

Reply:
Fixed.